contracts/coordinator/src/contract/migrations/v0_2_0.rs

```diff
@@ -0,0 +1,118 @@
+#![allow(deprecated)]
+
+use axelar_wasm_std::killswitch::State;
+use axelar_wasm_std::{killswitch, nonempty};
+use cosmwasm_std::{Addr, StdError, Storage};
+use cw2::VersionError;
+
+use crate::contract::CONTRACT_NAME;
+
+const BASE_VERSION: &str = "0.2.0";
+
+#[derive(thiserror::Error, Debug)]
+pub enum Error {
+    #[error(transparent)]
+    Std(#[from] StdError),
+    #[error(transparent)]
+    Version(#[from] VersionError),
+    #[error(transparent)]
+    NonEmpty(#[from] nonempty::Error),
+}
+
+pub fn migrate(storage: &mut dyn Storage, service_registry: Addr) -> Result<(), Error> {
+    cw2::assert_contract_version(storage, CONTRACT_NAME, BASE_VERSION)?;
+
+    killswitch::init(storage, State::Disengaged)?;
```

Comment:
I don't think the coordinator needs a killswitch, confirm with CJ. Since the coordinator only aggregates data and doesn't modify it itself I can't see a scenario where we would have to quickly deactivate it, but I can also see an argument for adding a killswitch to any contract

Reply:
He said he does not think we need it here. Removing...